homeassistant/components/light/__init__.py

```diff
@@ -241,7 +241,20 @@ def async_handle_light_service(service):
         if color_name is not None:
             params[ATTR_RGB_COLOR] = color_util.color_name_to_rgb(color_name)
 
+        xy_color = params.get(ATTR_XY_COLOR, None)
+
         for light in target_lights:
+            # Convert xy_color if unsupported
+            if xy_color is not None:
+                support_xy = bool(light.supported_features & SUPPORT_XY_COLOR)
+                brightness = params.get(ATTR_BRIGHTNESS, light.brightness)
```

Comment:
Exactly! It's quite feasible to want to change the color of a group of lights but maintain their individual brightness. Philips Hue also lets us set just the XY color and it will maintain current brightness so I feel the emulated XY should support that as well. The ```flux``` switch has ```disable_brightness_adjust``` that uses this.

Granted, it may be wrong sometimes, but not doing anything without a provided ```brightness``` will be wrong all the time.